### DISCUSSION TITLE
Talking thing

### DISCUSSION THREAD

**Discussion by olderatriangle** [LeoIsACoder/txt-thing-wip#3] [Ideas]
We can talk here and work on the thing.

**Comment by LeoIsACoder** [comment]
Gotcha

**Reply by olderatriangle** [reply]
So this is where we basically talk and make new ideas.

**Reply by olderatriangle** [reply]
I'm also listening to music while I'm working on this thing so yeah.

**Reply by olderatriangle** [reply]
Hello???

**Reply by LeoIsACoder** [reply]
Hey, Sorry I'm workiing on homework

**Reply by olderatriangle** [reply]
Oh ok.

**Reply by LeoIsACoder** [reply]
Math homework

**Reply by olderatriangle** [reply]
Thankfully, I have my book.

**Reply by LeoIsACoder** [reply]
I meant.

**Reply by olderatriangle** [reply]
Wait, I thought you said you were done.

**Reply by LeoIsACoder** [reply]
Can you unblock insta??

**Reply by LeoIsACoder** [reply]
or tell me how?

**Reply by olderatriangle** [reply]
It's a complicated process. I'm not sure you would understand.

**Reply by olderatriangle** [reply]
Here: https://github.com/Instagram Their github group.

**Reply by olderatriangle** [reply]
So um, I might have overdone it a bit with the website. Just look at it. https://leoisacoder.github.io/txt-thing-wip/

**Reply by LeoIsACoder** [reply]
Whspa

**Reply by olderatriangle** [reply]
Please put the website link in the description.

**Reply by olderatriangle** [reply]
I'm sorry I didn't get on in time. My mom didn't let me be on until 4.

**Reply by olderatriangle** [reply]
Anyways here. 1-760-728-6191

**Reply by olderatriangle** [reply]
Leo, I found it. https://www.mikedane.com/web-development/javascript/building-a-messaging-app/ This will help us build our messaging app. If it looks boring when we first make it. You and I can always freshen it up with our great friend, CSS.

**Reply by olderatriangle** [reply]
Here. [](https://www.khanacademy.org/computing/computer-programming/html-css-js/html-css-js-intro/v/welcome-to-making-webpages-interactive)

**Reply by olderatriangle** [reply]
https://www.khanacademy.org/computing/computer-programming/html-css-js/html-css-js-intro/v/welcome-to-making-webpages-interactive

**Reply by olderatriangle** [reply]
This is why I added JQuery to the repository. https://www.khanacademy.org/computing/computer-programming/html-js-jquery#jquery-intro

**Reply by olderatriangle** [reply]
But for now, stick with https://www.khanacademy.org/computing/computer-programming/html-css-js/html-css-js-intro/v/welcome-to-making-webpages-interactive

**Reply by olderatriangle** [reply]
Oh for more info, here. https://bit.ly/3bzBe5c
